components/automate-ui/src/app/pages/+compliance/shared/reporting/scan-results.service.ts

```diff
@@ -1,6 +1,10 @@
 import { Injectable } from '@angular/core';
 import { BehaviorSubject, Observable } from 'rxjs';
 
+interface ControlsByNode {
+  [nodeId: string]: any;
```

Comment:
So the controls do not have any definite type? I do see a lot of `any` values here and below. 🤷

Reply:
Yes, this entire area of the code really needs more types.